src/webrtc/groupCall.ts

```diff
@@ -609,6 +609,9 @@ export class GroupCall extends TypedEventEmitter<
             logger.log(
                 `groupCall ${this.groupCallId} setLocalVideoMuted stream ${this.localCallFeed.stream.id} muted ${muted}`,
             );
+
+            const stream = await this.client.getMediaHandler().getUserMediaStream(true, !muted);
```

Comment:
Since the audio flag is set to `true`, I take it that the code was already requesting audio, no matter what?

Reply:
Yeah, we basically always request audio since even when muted we want the track to always be there for quick toggling